tests/integration/test_views.py

```diff
@@ -37,7 +37,7 @@ def test_extract_journal_info(app_client):
     )
     assert response.status_code == 200
     assert "extracted_publication_infos" in response.json
-    assert len(response.json["extracted_publication_infos"]) == 1
+    assert len(response.json["extracted_publication_infos"]) == 2
```

Comment:
why not 1? should we add an empty `{}`?

Reply:
No, because we add an empty dict to the list of publication infos, and the length of publication info is 2